### PR TITLE
Feature/sqs

### PR DESCRIPTION
**Pull request by bashleigh**
## Brief description of what's changing
- [x] Moved http handler to http directory
- [x] created sqs handler 

## Brief description of why it's changed
Create a mock of sqs handler function

### CODE REVIEW

File: src/http/utils/json.parse.ts [hunk 1]
```diff
@@ -3,7 +3,8 @@ import { BadRequestException } from "../exceptions";
 
 export const JSONParse = (event: APIGatewayEvent): any => {
   let json: any;
-  const contentType = event?.headers['Content-Type'] || event?.headers['content-type'] || '';
+  const contentType =
+    event?.headers["Content-Type"] || event?.headers["content-type"] || "";
```
Comment (mthomasuk) [inline src/http/utils/json.parse.ts:7]: `event?.headers?["Content-type"]`

File: src/sqs/filter.unique.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { SQSRecord } from "aws-lambda";
+
+export type SqsFilterUniqueRecordsFunction = (
+  records: SQSRecord[],
+) => SQSRecord[];
+
+export const filterUniqueRecords: SqsFilterUniqueRecordsFunction = (
+  records: SQSRecord[],
+): SQSRecord[] => records;
```
Comment (mthomasuk) [inline src/sqs/filter.unique.ts:9]: doesn't this just return the argument?

Comment (bashleigh) [reply src/sqs/filter.unique.ts:9]: yea it's an optional function that doesn't require a default. Maybe I should make it an optional call also

File: src/http/utils/json.parse.ts (hunk 1 above)
Comment (bashleigh) [reply src/http/utils/json.parse.ts:7]: wanna add it?

File: src/test/events/sqs.event.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import {SQSEvent} from 'aws-lambda';
```
Comment (mthomasuk) [inline src/test/events/sqs.event.ts:1]: need consistent spacing in imports

File: src/test/sqs/sqs.handler.spec.ts [hunk 4]
```diff
@@ -0,0 +1,169 @@
+import { RecordResults, SQSHandle, SQSHandler } from "../../sqs";
+import AWS, { SQS } from 'aws-sdk';
+import { mockSQSEvent } from "../events/sqs.event";
+import mockContext from "aws-lambda-mock-context";
+import { SQSRecord } from "aws-lambda";
```
Comment (mthomasuk) [inline src/test/sqs/sqs.handler.spec.ts:5]: need consistent ordering of imports

File: src/http/utils/json.parse.ts (hunk 1 above)
Comment (mthomasuk) [reply src/http/utils/json.parse.ts:7]: what do you mean?

File: src/sqs/filter.unique.ts (hunk 2 above)
Comment (mthomasuk) [reply src/sqs/filter.unique.ts:9]: gotcha

Comment (mthomasuk) [reply src/sqs/filter.unique.ts:9]: could leave a comment above it

File: package.json [hunk 5]
```diff
@@ -53,5 +55,10 @@
   "dependencies": {
     "lodash.merge": "^4.6.2"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn format"
```
Comment (mthomasuk) [inline package.json:61]: yarn add -D pretty-quick

"pre-commit": "pretty-quick --staged"

File: src/http/utils/json.parse.ts (hunk 1 above)
Comment (bashleigh) [reply src/http/utils/json.parse.ts:7]: wanna open a PR?

**Pull request merged**